### PR TITLE
Fix duplicated selected AccountCache

### PR DESCRIPTION
**Pull request by hoangdat**

### CODE REVIEW

**Review by tddang-linagora — APPROVED** [review]

File: lib/features/login/data/local/account_cache_manager.dart [hunk 1]
```diff
@@ -35,8 +35,8 @@ class AccountCacheManager {
         .removeDuplicated()
         .whereNot((account) => account.accountId == newAccountCache.accountId)
         .toList();
+      await _accountCacheClient.clearAllData();
```
Comment (dab246) [inline lib/features/login/data/local/account_cache_manager.dart:38]: why do that?

Comment (hoangdat) [reply lib/features/login/data/local/account_cache_manager.dart:38]: In case of newAllAccount empty, you not update the cache.

Comment (dab246) [reply lib/features/login/data/local/account_cache_manager.dart:38]: So what effect does it have on the current cache, if we follow the old flow? Please describe more clearly the root cause of this problem.

**Review by dab246 — APPROVED** [review]
LGTM

**Pull request merged**